src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -123,5 +128,24 @@ object Boot extends App {
     }
   }
 
+  /**
+   * Enables the rawls service account in ldap. Allows service to service auth through the proxy.
+   * @param gcsDAO
+   * @param userDirDAO
+   */
+  def enableServiceAccount(gcsDAO: HttpGoogleServicesDAO, userDirDAO: JndiUserDirectoryDAO): Unit = {
+    val enableServiceAccountFuture = for {
+      serviceAccountUser <- gcsDAO.getServiceAccountRawlsUser()
+      _ <- userDirDAO.createUser(serviceAccountUser).recover { case e: NameAlreadyBoundException => Unit } // if it already exists, ok
+      _ <- userDirDAO.enableUser(serviceAccountUser).recover { case e: AttributeInUseException => Unit } // if it is already enabled, ok
+    } yield Unit
+
+    enableServiceAccountFuture.onFailure {
+      // this is logged as a warning because almost always the service account is already enabled
```

Comment:
judgement call really... if an error happens here, the only time it really matters *at this point in the code* is when a new service account is in use.